changes/23679-optimize-software-versions

```diff
@@ -0,0 +1 @@
+* Added `without_vulnerability_details` to software versions endpoint so the CVE details can be truncated
```

Comment:
```suggestion
* Added `without_vulnerability_details` to software versions endpoint so CVE details can be truncated when on Fleet Premium
```

Also go ahead and mention the URL pattern of the endpoint, similar to what I did for the same thing on the hosts list endpoint.

Reply:
sounds much better. thanks!